wiki/osu!_wiki/Contribution_guide/Best_practices/vi.md

```diff
@@ -78,22 +48,55 @@ Nếu bạn gặp vấn đề gì khi đang sử dụng công cụ GitHub hoặc
 
 4. Nhấn nút `Run Workflow` và chờ nó hoàn thành. Nếu bạn tò mò xem công cụ này hoạt động ra sao, nhấn lại vào `Sync from osu! upstream`.
```

Comment:
missing an image

for reference:
```md
4. Click the `Run Workflow` button and wait for the workflow to complete. If you're curious about how the tool works, click on the `Sync from osu! upstream` workflow task.

![](img/github-actions-workflow-overview.png "GitHub Actions Workflow - Workflow Overview")
```

Reply:
i cant put an alternative text sit next to an image?

Reply:
i think i would follow the English article, and not put the alt-text to that image

Reply:
ok i was a little stupid so i thought i was messing the other stuff
added